README.md

```diff
@@ -20,6 +20,7 @@ MIOpen supports two programming models -
 * [Half](http://half.sourceforge.net/) - IEEE 754-based half-precision floating point library
 * [Boost](http://www.boost.org/) at version 1.72 (other versions are not supported)
   * MIOpen uses `boost-system` and `boost-filesystem` packages to enable persistent [kernel cache](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/cache.html)
+* [libsqlite3-dev](https://sqlite.org/index.html) package which can be automatically installed with `cmake -P install_deps.cmake --minimum `
```

Comment:
This really should only have a description of the dependency: `for reading and writing performance database tunings`. It also should be name `Sqlite3`.

We already document automatically installing the dependencies below nor we do repeat this for other dependencies such as boost.

Reply:
thanks for the quick review @pfultz2  I will update the text.